server/middleware/products.coffee

```diff
@@ -16,6 +16,7 @@ get = wrap (req, res) ->
     )
     return res.send(products)
 
+  # Remove old unsupported subscription products
   products = _.filter products, (p) -> p.name not in ['year_subscription', 'lifetime_subscription2']
```

Comment:
whitelist seems easier to me.  Then we'd only need to keep track of what is currently supported rather than slowly building up large lists of past features via blacklists.